### PR TITLE
Changes for Calendars.ReadBasic permission in v1

### PR DESCRIPTION
**Pull request by Harini84**
Documentation changes for introducing Calendars.ReadBasic permission in v1 endpoint

### CODE REVIEW

File: changelog/Microsoft.Exchange.json [hunk 1]
```diff
@@ -2117,6 +2133,144 @@
...
+          "ChangeType": "Addition",
+          "Description": "Added support for the application and delegated permission `Calendars.ReadBasic` to the [List instances](https://learn.microsoft.com/en-us/graph/api/event-list-instances?view=graph-rest-1.0) method of the [event](https://learn.microsoft.com/en-us/graph/api/resources/event?view=graph-rest-1.0) resource.",
+          "Target": "event"
+        },
+        {
+          "Id": "ae497d52-a781-46de-bec7-12a215043afd",
+          "ApiChange": "Permission",
+          "ChangedApiName": "Calendars.ReadBasic",
+          "ChangeType": "Addition",
+          "Description": "Added support for the application and delegated permission `Calendars.ReadBasic` to the [List calendarGroups](https://learn.microsoft.com/en-us/graph/api/user-list-calendargroups?view=graph-rest-1.0) method of the [user](https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0) resource.",
```
Comment (Jarbas-MSFT) [inline changelog/Microsoft.Exchange.json:2232]: Please, update this PR to include an update to the [List calendarGroups](https://learn.microsoft.com/en-us/graph/api/user-list-calendargroups?view=graph-rest-1.0) method.

File: changelog/Microsoft.Exchange.json [hunk 2]
```diff
@@ -2046,6 +2046,22 @@
           "Description": "Added support for the application and delegated permission `Calendars.ReadBasic` to the [List calendars](https://learn.microsoft.com/en-us/graph/api/calendargroup-list-calendars?view=graph-rest-beta) method of the [calendarGroup](https://learn.microsoft.com/en-us/graph/api/resources/calendargroup?view=graph-rest-beta) resource.",
           "Target": "calendarGroup"
         },
+        {
```
Comment (Jarbas-MSFT) [inline changelog/Microsoft.Exchange.json:2049]: FYI: Added missing entries for the corresponding BETA PR.

**Comment by Jarbas-MSFT** [comment]
Hello @Harini84, I reviewed your PR and left some comments. Please, take a look when you will have time.

File: api-reference/v1.0/api/calendar-get.md [hunk 3]
```diff
@@ -24,7 +24,7 @@ Depending on the type of calendar that the event is in and the permission type (
 
 | Calendar | Delegated (work or school account) | Delegated (personal Microsoft account) | Application |
 |:-----|:-----|:-----|:-----|
-| user calendar | Calendars.Read, Calendars.ReadWrite | Calendars.Read, Calendars.ReadWrite | Calendars.Read, Calendars.ReadWrite |
+| user calendar | Calendars.ReadBasic, Calendars.Read, Calendars.ReadWrite | Calendars.ReadBasic, Calendars.Read, Calendars.ReadWrite | Calendars.ReadBasic, Calendars.Read, Calendars.ReadWrite |
```
[the remark this answers is not in the document]
Comment (Harini84) [reply api-reference/v1.0/api/calendar-get.md:27]: @Jarbas-MSFT 
Calendars.ReadBasic is the name and it is the only new permission introduced in Calendar. There is no Calendars.ReadBasic.All that exists today.
Previous owner has updated this doc to reflect Calendars.ReadBasic: 
(https://learn.microsoft.com/en-us/graph/permissions-reference#application-permissions-11)

image

Can you please help in updating Calendars.ReadBasic.All to Calendars.ReadBasic in the file you are referring?
I am not aware in which *.md file these details are to be updated. Kindly share the details.

File: changelog/Microsoft.Exchange.json (hunk 2 above)
Comment (Harini84) [reply changelog/Microsoft.Exchange.json:2049]: Thank you!

File: changelog/Microsoft.Exchange.json (hunk 1 above)
Comment (Harini84) [reply changelog/Microsoft.Exchange.json:2232]: Updated and added in line 2180

File: api-reference/v1.0/api/calendar-get.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Jarbas-MSFT) [reply api-reference/v1.0/api/calendar-get.md:27]: Hi @Harini84,

> Can you please help in updating Calendars.ReadBasic.All to Calendars.ReadBasic in the file you are referring?

OK, thanks for the clarification. However, this is still a confusion and maybe even a problem for our customers. The "file" I am referring in that image is a screenshot from the API permissions section in App registration in Azure Active Directory from where customers can add new permissions to their AAD apps. So, the "Calendars.ReadBasic.All" permission has been deployed as an application permission. If the application permission in this case should be only "Calendars.ReadBasic", then this is not something we can fix via a markdown update, rather this is something that you need to follow up with another team internally. 

> Previous owner has updated this doc to reflect Calendars.ReadBasic:

Can you please share with me a pointer to the PR that contains this API? The screenshot you shared with me is related to "**delegated**" permissions, but the issue we have is with an "**application**" permission.

**Comment by Jarbas-MSFT** [comment]
Hi @Harini84, I checked your PR again and left two comments. Please take a look when you will have time:
- https://github.com/microsoftgraph/microsoft-graph-docs/pull/21614#discussion_r1232961889
- https://github.com/microsoftgraph/microsoft-graph-docs/pull/21614#discussion_r1232963672

File: api-reference/v1.0/api/calendar-get.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Harini84) [reply api-reference/v1.0/api/calendar-get.md:27]: @Jarbas-MSFT 
This is the API documentation PR raised by previous owner:
https://github.com/microsoftgraph/microsoft-graph-docs/pull/19209

This is my PR for beta documentation changes:
https://github.com/microsoftgraph/microsoft-graph-docs/pull/20282

Please update if any further information needed to change app permission reference from 'Calendars.ReadBasic.All' to 'Calendars.ReadBasic'.

File: changelog/Microsoft.Exchange.json (hunk 1 above)
Comment (Harini84) [reply changelog/Microsoft.Exchange.json:2232]: @Jarbas-MSFT 
Can you please clarify if change is to be made in Microsoft.Exchange.json file or in user-list-calendargroups.md file.

The first comment refers in Microsoft.Exchange.json but the last comment refers to user-list-calendargroups.md file. Please clarify.

**Comment by JarbasHorst** [comment]
Hi @Harini84, I addressed [this](https://github.com/microsoftgraph/microsoft-graph-docs/pull/21614#discussion_r1235045736) comment directly with my PR [21970](https://github.com/microsoftgraph/microsoft-graph-docs/pull/21970).

Please, resolve the merge conflicts.

Hi @Lauragra, this PR is ready for final review.

**Comment by Harini84** [comment]
@Jarbas-MSFT 
Pulled the main branch and merge with this branch is complete.
Please update if any further modifications required from my end.

**Comment by JarbasHorst** [comment]
@Lauragra this PR is ready for final review. Please, merge PR [22019](https://github.com/microsoftgraph/microsoft-graph-docs/pull/22019) into this branch to include my final edits.

**Pull request merged**